_locales/en/messages.json

```diff
@@ -154,5 +154,32 @@
   },
   "search": {
     "message": "Search"
+  },
+  "permissionsTitle": {
+    "message": "Permissions - Scratch Addons"
+  },
+  "permissions": {
+    "message": "Permissions"
+  },
+  "enablePermissionsTitle": {
+    "message": "Please enable site access"
+  },
+  "enablePermissionsDescription": {
+    "message": "Scratch Addons needs site access permission to run addons."
+  },
+  "enableButton": {
+    "message": "Enable"
+  },
+  "enableButtonDescription": {
+    "message": "Click the button to enable."
+  },
+  "permissionsScreenAlt": {
+    "message": "Screenshot of a dialog to enable site access"
```

Comment:
a11y is good, but I don't think this description will actually help a user with bad vision, since it's a screenshot